heartbeat/docs/monitors/monitor-browser.asciidoc

```diff
@@ -189,7 +189,19 @@ run with 5Mbps download, 3Mbps upload and 20ms latency which emulates a standard
 Cable connection.
 
 Users can control the throttling parameters, Below is an example of emulating a
-3G connection with 1.6Mbps download, 750Kbps upload and 150ms round trip time.
+3G connection with 1.6Mbps download, 750Kbps upload and 150ms round trip time
+in two different supported formats.
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: browser
+  schedule: '@every 1m'
+  throttling: "1.6d/0.75u/150l"
```

Comment:
IMO, we should remove the old format and references so we can do breaking change on the Synthetics agent from next version. Would give us enough room to move to GA. WDYT?

Reply:
I'm not nuts about removing it, though I'd be fine with deprecating it, which we can do in a follow-up.

I realize we're in beta so we could just remove it, but for people who are using it that's rude, and it's esp. tricky for configs serialized in fleet. I think we need to have discuss what changing config options looks like in our new world of GUI configs.

Reply:
@andrewvc I know this has already been merged, but the _new format_ seems to still show the old one too (in `throttling: "1.6d/0.75u/150l"`), shouldn't the new one be:

```
throttling:
  download: 1.6
  upload: 0.75
  latency: 150
```

Reply:
Currently, HB supports both formats and we have not deprecated the old one which could be the reason.

Reply:
I've created a PR to fix this, good catch @paulb-elastic https://github.com/elastic/beats/pull/31738/files